test/helix.proj

```diff
@@ -0,0 +1,51 @@
+<Project DefaultTargets="Test">
+
+  <!-- Version included until we get global.json generation to support this SDK. -->
+  <Sdk Name="Microsoft.DotNet.Helix.Sdk" Version="2.0.0-beta.19073.6" />
+
+  <Target Name="Gather" BeforeTargets="Test">
+    <ItemGroup>
+      <ProjectsToTest Include="$(ProjectsToTest)" />
+    </ItemGroup>
+    <MSBuild Projects="@(ProjectsToTest)"
+              Targets="CreateHelixPayload"
+              BuildInParallel="false">
```

Comment:
Why is this set to false? Doesn't it make publish time very-very long?

Reply:
Not sure here, but if someone does this intentionally it's usually because debugging or fixing the concurrency issues they hit is more expensive than setting the property.  Note that zipping up payloads and uploading them is usually fast (build / restore being much slower)

Reply:
Currently, it takes >2h to publish all the test projects. I would say it's a major blocker for usability.

Reply:
I'll try flipping it back to true, I don't remember anymore what the issue was that was fixed by setting this false, we'll see if its still there shortly :)